agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/AiConfigCustomizer.java

```diff
@@ -148,7 +148,12 @@ private static void enableInstrumentations(Configuration config, Map<String, Str
     properties.put("otel.instrumentation.liberty.enabled", "true");
     properties.put("otel.instrumentation.liberty-dispatcher.enabled", "true");
     properties.put("otel.instrumentation.log4j-appender.enabled", "true");
-    properties.put("otel.instrumentation.logback-appender.enabled", "true");
+
+    String logbackSystemProperty =
+        System.getProperty("otel.instrumentation.logback-appender.enabled");
+    if (logbackSystemProperty == null || Boolean.parseBoolean(logbackSystemProperty)) {
+      properties.put("otel.instrumentation.logback-appender.enabled", "true");
+    }
```

Comment:
let's pass `otelConfig` to this method and then can use:

```suggestion
    properties.put("otel.instrumentation.logback-appender.enabled", otelConfig.getBoolean("otel.instrumentation.logback-appender.enabled", true));
```